client/helpers/render-elements.helpers.tsx

```diff
@@ -26,49 +30,70 @@ const SIMPLE_TYPES = [
 // array of types such as banners, info, headings etc.
 const NOT_TO_BE_RENDERED = ['button', 'htmlelement', 'simplebuttonadvanced', 'simplecontent'];
 
+const MISC_LABELS_TO_REMOVE = ['Text/Images'];
+
+const checkIfGridHasValues = (gridData: any) => {
+  let hasValues = false;
+
+  gridData.forEach((value: any) => {
+    hasValues = !!Object.values(value).find((item: any) => !!item);
+    if (hasValues) return true;
```

Comment:
return in a `forEach` doesn't exit the loop, we only need to check if theres at least 1 truthy value no? I think this will go through all the values

what about something like this?

```
for (const value of gridData) {
    if (Object.values(value).some((item: any) => !!item)) {
      hasValues = true;
      break;
    }
  }
```

that way it will exit on first truthy value